version

```diff
@@ -1 +1 @@
-3.3.1
+3.3.2-dirty.24.6a3540a
```

Comment:
you'll want to revert this change to prevent an accidental version bump. We might want to brainstorm some solutions on keeping this from accidentally updating; I got caught by this a number of times too

Reply:
yep, will do when this PR Is done. I have an issue right now where the yaw, roll, and pitch functions aren't working properly so I'm going to work that out in a bit.